lib/kapnismology/terminal.rb

```diff
@@ -1,20 +1,29 @@
 module Kapnismology
   class Terminal
-
     def self.green(msg)
-      "\e[32m\e[1m#{msg}\e[0m"
+      colorize("\e[32m", msg)
     end
 
     def self.red(msg)
-      "\e[31m\e[1m#{msg}\e[0m"
+      colorize("\e[31m", msg)
     end
 
     def self.yellow(msg)
-      "\e[33m\e[1m#{msg}\e[0m"
+      colorize("\e[33m", msg)
     end
 
     def self.bold(msg)
-      "\e[1m#{msg}\e[0m"
+      colorize('', msg)
```

Comment:
Not all messages are bold, but all colored messages are bold 
<img width="613" alt="screen shot 2016-08-12 at 10 27 00 am" src="https://cloud.githubusercontent.com/assets/1803073/17610270/55d69894-6077-11e6-8a8e-5d3d97cb0182.png">